.github/workflows/ci-kubernetes-itests.yml

```diff
@@ -0,0 +1,108 @@
+name: Quarkus Kubernetes CI
+
+env:
+  MAVEN_ARGS: -B -e
+
+on:
+  push:
+    branches:
+      - main
+  pull_request_target:
+    types: [ opened, edited, labeled, reopened, synchronize ]
+
+jobs:
+  cache:
+    name: Cache
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+      - name: Cache .m2 registry
+        uses: actions/cache@v2.1.5
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}-${{ github.sha }}
+          restore-keys: ${{ runner.os }}-maven-
+      - name: Install artifacts
+        run: mvn -f pom.xml ${MAVEN_ARGS} -DskipTests clean install -pl :quarkus-integration-test-kubernetes-invoker -am
+  kubernetes:
+    name: Kubernetes Integration Tests
+    needs: cache
+    if: (github.event.label.name == 'area/kubernetes' || contains(github.event.pull_request.labels.*.name, 'area/kubernetes')) &&  ${{ secrets.QUAY_E2E_USERNAME}} != null }} && ${{ secrets.QUAY_E2E_PASSWORD != null }}
```

Comment:
Is `github.event.label.name` the one when you add a label? If so, I would make the test a bit more complicated to only trigger a build for `labeled` if it's the `area/kubernetes` added. Otherwise, adding the backport label for instance will trigger another build. We don't want this.

Reply:
Just to be extra clear, I'm talking about this case:
- the PR already has the `area/kubernetes` label
- we add the `triage/backport?` label

We don't want this case to trigger the Kubernetes CI.

Reply:
Good point